proto/prysm/v1alpha1/blobs.pb.gw.go

```diff
@@ -1,4 +1,3 @@
-//go:build ignore
 // +build ignore
 
-package ignore
+package ignore
```

Comment:
What happened here?

Reply:
i think it got added on regeneration i'll fix it